src/django/oar/settings.py

```diff
@@ -147,6 +147,9 @@
 
 AUTH_USER_MODEL = 'api.User'
 
+CSRF_COOKIE_SECURE = True
+SESSION_COOKIE_SECURE = True
+
```

Comment:
Note, that there is no need to adjust **DEBUG** condition here for the localhost because it [works](https://developer.mozilla.org/en-US/docs/Web/HTTP/Guides/Cookies#block_access_to_your_cookies) locally out of the box.
> It's never sent with unsecured HTTP (except on localhost)